options.c

```diff
@@ -80,6 +95,19 @@ int init_options(int opt_cnt, char *opt_arr[], struct tool_options *options)
 	return rc;
 }
 
+static int verify_target(struct tool_options *options, int target)
```

Comment:
in case of read, you don't need to know if its a peer or local - you are reading both.
 I think you should at least make a comment of it.
 
 If unipro.h is already included, you can add to this patch all further checks,
 e.g. of the unipro idn etc.